### DISCUSSION TITLE
TokenId-->property_version 的值会被更改吗？

### DISCUSSION THREAD

**Discussion by livelybug** [NonceGeek/Move-Camp#282] [Q&A] [answered]
在看 <05. NFT 合约案例讲解>
`TokenId-->property_version` 的值会被更改吗？
我搜了一下，发现一直都是默认值，没被改过

![image](https://user-images.githubusercontent.com/13627060/217710630-f7274f8a-db49-4146-9f03-1953c5bd9f32.png)

我的理解是，创建NFT时，property version是通过
[let cur_property_version = token_data.largest_property_version + 1;](https://github.com/aptos-labs/aptos-core/blob/9518e1be2ff471cc875385ba436132c6ead46732/aptos-move/framework/aptos-token/sources/token.move#L819)
递增的 。但`TokenId-->property_version` 有什么用呢？

**Comment by hqwangningbo** [comment, marked as answer]
会修改的，仔细看下这里，其实这里的cur_property_version就是在修改property_version
<img width="789" alt="image" src="https://user-images.githubusercontent.com/57781136/217714316-89f0d40a-d740-458e-a152-2affb05de9a3.png">

**Reply by livelybug** [reply]
问了个傻问题 :<
